sigs.yaml

```diff
@@ -1562,7 +1562,7 @@ sigs:
       company: Dell
     - github: salaxander
       name: Xander Grzywinski
-      company: Defense Unicorns
```

Comment:
When you alter company affiliation you have to rerun `make generated` for the README files for the groups you're listed in 😅 

(Also I think we usually use "Independent" in this case)